roles/sap_hana_preconfigure/tasks/sapnote/2684254/configuration.yml

```diff
@@ -66,15 +66,15 @@
 - name: "Energy Performance Bias (EPB, applies to Intel-based systems only)"
   ansible.builtin.lineinfile:
     path: /etc/init.d/boot.local
-    mode: 0744
+    mode: "0744"
```

Comment:
Why not execution?

Reply:
We have mode `0755` for this file defined here:
https://github.com/sap-linuxlab/community.sap_install/blob/c1b479ef710f0f0803a5a081eafd621465c86d4b/roles/sap_hana_preconfigure/tasks/RedHat/generic/disable-ksm.yml#L8-L12

Also from looking at my RHEL test systems, it seems that this is the default for RHEL. So likely it is also the default for SLES.